services/app-api/src/postgres/prismaTypes.ts

```diff
@@ -1,20 +1,42 @@
-import { PrismaClient, UpdateInfoTable, User } from '@prisma/client'
+import {
+    ActuaryContact,
+    ContractDocument,
+    ContractRevisionTable,
+    ContractSupportingDocument,
+    FederalAuthority,
+    ManagedCareEntity,
+    PrismaClient,
+    RateRevisionsOnContractRevisionsTable,
+    RateRevisionTable,
+    StateContact,
+    UpdateInfoTable,
+    User,
+} from '@prisma/client'
 
 // This is the type returned by client.$transaction
 type PrismaTransactionType = Omit<
     PrismaClient,
     '$connect' | '$disconnect' | '$on' | '$transaction' | '$use'
 >
 
-// For use in prisma queries, include the updater in an updateInfo
-const updateInfoIncludeUpdater = {
-    include: {
-        updatedBy: true,
-    },
+type ContractRevisionTableWithRelations = ContractRevisionTable & {
+    submitInfo?: UpdateInfoTableWithUpdater | null
+    unlockInfo?: UpdateInfoTableWithUpdater | null
+    stateContacts: StateContact[]
+    addtlActuaryContacts: ActuaryContact[]
```

Comment:
@macrael @JasonLin0991 🤔 Do we think it could cause any issues putting a field thats so clearly about the rates on the contract still? I'm thinking about rates only dashboard and views - are we thinking they would be editing additional actuary contacts there tool?

Reply:
I remember asking design about this field a while ago if `addtlActuaryContacts` applied to the rate or the contract as a whole. I remember them saying it was applied to the contract as a whole, we should confirm and bring this topic up.

My questions if we did move this to the rates:
- Would we move displaying `addtlActuaryContacts` to the Rate details page under each rate?
   - This would make sense to do if we are separating `addtlActuaryContacts`.
   - Would `addtlActuaryContacts`  always be the same for each rate? Lets say a contract has a shared rate, would the `addtlActuaryContacts` ever be different?
- If we don't move it, then how do we compose `addtlActuaryContacts`? Do we merge all the rate certifications `addtlActuaryContacts`?
   - What about shared rates, do their `addtlActuaryContacts` apply to the contract?

Reply:
Sounds like we are going to move it from today's DTBM. I will take care of this in my `insertRate` ticket from the data side.